examples/buildkit.yaml

```diff
@@ -0,0 +1,30 @@
+# Example to use Buildkit
+# $ limactl start ./buildkit.yaml
+
+# To run `buildkit` on the host (assumes buildctl is installed):
+# $ export BUILDKIT_HOST=unix://$HOME/.lima/buildkit/buildkitd.sock
+# $ buildctl debug workers
+
+images:
+- location: "~/Downloads/impish-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "~/Downloads/impish-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+- location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+mounts:
```

Comment:
are these mounts even needed ? normally everything would be in the build context, no ?

Reply:
yes you are right, we don't need to specify them here.